.github/workflows/ci.yml

```diff
@@ -411,6 +411,56 @@ jobs:
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
 
+  test-iceberg:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: 11
+          cache: 'maven'
+      - name: Configure Problem Matchers
+        run: |
+          echo "::add-matcher::.github/problem-matcher.json"
+      - name: Maven Install
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          $RETRY $MAVEN install ${MAVEN_FAST_INSTALL} -am -pl :trino-iceberg
+      - name: Run Iceberg Tests
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESSKEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRETKEY }}
+          AWS_REGION: "us-east-2"
+          S3_BUCKET: "presto-ci-test"
+        run: |
+          if [ "${AWS_ACCESS_KEY_ID}" != "" ]; then
```

Comment:
This should be moved as an additional step to the `test` job and it should be conditional, by adding `if:  env.AWS_ACCESS_KEY_ID != ''`. This will allow running the whole ci in forks without this step failing.

Reply:
See how I did this for memsql and bigquery in https://github.com/trinodb/trino/pull/10323/commits/9fe62826f27c29953ccc0f539ad5f9fb0c92bfa2

Reply:
>  This will allow running the whole ci in forks without this step failing.

that's the obvious upside. the downside is that a ci typo disables a test.
anyway; pre-existing, not of a concern for this PR specifically